<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelCamera">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>431</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTenNguoiMoi">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nhập tên của bạn:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChupAnh">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Chụp ảnh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDongY">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Đồng ý</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHuy">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>270</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hủy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
